src/libraries/System.Formats.Tar/src/System/Formats/Tar/GnuTarEntry.cs

```diff
@@ -52,47 +57,33 @@ public GnuTarEntry(TarEntry other)
                 _header._cTime = gnuOther.ChangeTime;
                 _header._gnuUnusedBytes = other._header._gnuUnusedBytes;
             }
-            else
+            else if (other is PaxTarEntry paxOther)
```

Comment:
I wonder if this is what folks expect?  If we know that sometimes it won't work for gnu entries, and PAX always sets these, maybe we should omit by default.

Reply:
I was hesitant to lose data, but you're right, we would end up creating a GNU entry that might be problematic. We better _not_ set atime and ctime when converting from pax to gnu, and make sure to document this "loss of data" clearly.

Reply:
It seems that creating GNU entries with these set is opt-in for other tools too, so I think we should try to make it opt-in in our API.  So when users do things that aren't explicitly trying to apply these to Gnu entries we don't do so automatically.  If we get feedback that manually copying the timestamps is burdensome, perhaps we could add another constructor that has a flag to indicate the user wishes to copy any timestamp info.